motor/motorid/awsec2/metadata_local.go

```diff
@@ -29,18 +30,26 @@ func (m *LocalEc2InstanceMetadata) Identify() (Identity, error) {
 	if err != nil {
 		return Identity{}, err
 	}
-	filters := []ec2types.Filter{
-		{
-			Name:   aws.String("resource-id"),
-			Values: []string{doc.InstanceID},
-		},
-	}
-	tags, err := ec2svc.DescribeTags(ctx, &ec2.DescribeTagsInput{Filters: filters})
 	name := ""
+	// try and fetch this from the metadata, if the tag metadata service is enabled.
+	nameTag, err := m.getMetadataValue(metadata, "tags/instance/Name")
```

Comment:
In the previous PR, we had `m.instanceNameTag()` in `metadata_cmd.go`. Is it possible to combine them somehow? Now, we have two different approaches to get to the same tag.

Reply:
The other one (the remote) relies on curl as we cannot create a metadata client when executing remotely, that would instantiate the client on our own machine rather than the ec2 one.  I can use curl here too but I prefer this as it's got also better err handling